internal/apis/certmanager/validation/certificaterequest_test.go

```diff
@@ -610,8 +610,10 @@ func TestValidateCertificateRequest(t *testing.T) {
 					Usages:    []cminternal.KeyUsage{cminternal.UsageKeyEncipherment, cminternal.UsageDigitalSignature},
 				},
 			},
-			a:     someAdmissionRequest,
-			wantE: []*field.Error{},
+			a: someAdmissionRequest,
+			wantE: []*field.Error{
+				field.Invalid(fldPath.Child("request"), nil, "encoded CSR error: the KeyUsages [] do not match the expected KeyUsages [ 'digital signature', 'key encipherment' ]"),
```

Comment:
This behavior is more correct, if a empty set of KeyUsages is encoded, we still want to compare it to the Usages array,